packages/react-native/Libraries/Renderer/implementations/ReactFabric-dev.js

```diff
@@ -7,9 +7,6 @@
  * @noflow
  * @nolint
  * @preventMunge
- * @generated SignedSource<<6b3694149dd8856f0c741d3d94e05646>>
- *
- * This file was sync'd from the facebook/react repository.
```

Comment:
The fact that these are not signed is not great... that's a step that the script does... I can try and sign them manually tomorrow.